backend/FwHeadless/Program.cs

```diff
@@ -77,6 +86,30 @@
     logger.LogDebug("crdtFile: {crdtFile}", crdtFile);
     logger.LogDebug("fwDataFile: {fwDataFile}", fwDataProject.FilePath);
 
+    var fwdataApi = SetupFwData(fwDataProject, srService, projectCode, logger, fwDataFactory);
+    var crdtProject = await SetupCrdtProject(crdtFile, projectLookupService, projectId, projectsService, projectFolder, fwdataApi.ProjectId, config.Value.LexboxUrl);
+
+    var miniLcmApi = await services.OpenCrdtProject(crdtProject);
+    var crdtSyncService = services.GetRequiredService<CrdtSyncService>();
+    await crdtSyncService.Sync();
+
+
+    var result = await syncService.Sync(miniLcmApi, fwdataApi, dryRun);
+    logger.LogInformation("Sync result, CrdtChanges: {CrdtChanges}, FwdataChanges: {FwdataChanges}", result.CrdtChanges, result.FwdataChanges);
+    fwDataFactory.CloseProject(fwDataProject);
```

Comment:
Does this maybe belong in a finally block?

Reply:
yeah it should, I was thinking of making a `using` statement to ensure it gets cleaned up, I forgot about the guarantee that offers lol.